nexus/src/external_api/http_entrypoints.rs

```diff
@@ -6386,7 +6389,7 @@ async fn timeseries_schema_list(
 async fn timeseries_query(
     rqctx: RequestContext<ApiContext>,
     body: TypedBody<params::TimeseriesQuery>,
-) -> Result<HttpResponseOk<Vec<oximeter_db::oxql::Table>>, HttpError> {
+) -> Result<HttpResponseOk<OxqlQueryResult>, HttpError> {
```

Comment:
Could you use `views::OxqlQueryResult` here?

Reply:
Yep, done in 42aa9cad6